### ISSUE TITLE
suit storage units can not be properly locked with the locking feature

### ISSUE THREAD

**Issue by Spacedong** [HippieStationCode/HippieStation13#2788]
any person, ID or not, can just press the Unlock button to unlock them, regardless of the person with the ID who locked it. it's fucking stupid and basically go fuck yourself /tg/

**Comment by Pyko1** [comment]
Intended, its to lock the person inside after you have stuffed him in.

**Comment by Pyko1** [comment]
very much like dna scanners

**Comment by Spacedong** [comment]
its not intended though, go look at mcbawg's explanation

**Comment by Pyko1** [comment]
https://github.com/HippieStationCode/HippieStation13/blob/5462010e5bcc8a61ad5618ffd0f04f07eeb570b0/code/game/machinery/suit_storage_unit.dm

Codewise literally the only reason it seems to have is to keep the occupant inside after you hack the machine.

**Comment by Spacedong** [comment]
yeah it was removed by /tg/ because they can't code properly, its supposed to be locked via ID
